apps/src/templates/teacherHomepage/Courses.jsx

```diff
@@ -0,0 +1,78 @@
+import $ from 'jquery';
+import React from 'react';
+import RecentCoursesCollapsible from './RecentCoursesCollapsible';
+import CollapsibleSection from './CollapsibleSection';
+import GradientNavCard from './GradientNavCard';
+import shapes from './shapes';
+import i18n from "@cdo/locale";
+
+const Courses = React.createClass({
+  propTypes: {
+    courses: shapes.courses,
+    englishTeacher: React.PropTypes.bool.isRequired
+  },
+
+  componentDidMount() {
+    if (this.props.englishTeacher) {
+      $('.courseexplorer').appendTo(this.refs.courseExplorer).show();
```

Comment:
Also worth providing a comment here about where coursexplorer, etc. are coming from.

Also, should those be ids instead of classNames? (If there's just a single instance, presumably yes).